astropy/table/table.py

```diff
@@ -494,6 +494,15 @@ def loc(self):
         return TableLoc(self)
 
     @property
+    def loc_indices(self):
+        '''
```

Comment:
Use `"""` not `'''` for multi-line strings (at least within `table`, `time`, and `io.ascii` subpackages).